src/com/carolinarollergirls/scoreboard/core/impl/BoxTripImpl.java

```diff
@@ -83,6 +83,14 @@ protected void valueChanged(PermanentProperty prop, Object value, Object last, F
                 ((Fielding)f).set(Fielding.Value.PENALTY_BOX, false);
             }
         }
+        if ((prop == Value.END_FIELDING || prop == Value.END_AFTER_S_P || prop == Value.END_BETWEEN_JAMS)
```

Comment:
Should we use recalculated here rather than hand-coding this?

Reply:
This is not a good fit for recalculate:
1. The fielding does not know if it's the start or end of any BoxTrip, so we would need nontrivial logic to maintain the listeners.
2. updateBoxTripSymbols() updates 3 values at once. That doesn't fit well into computeValue. And going from computeValue to a function that will in turn set the value requires fiddling around with flags.

So overall this solution is a lot cleaner.